common/src/main/java/com/wynntils/features/EmeraldPouchHotkeyFeature.java

```diff
@@ -0,0 +1,197 @@
+                                        player.inventoryMenu.getStateId(),
+                                        slotNumber,
+                                        1,
+                                        ClickType.PICKUP,
+                                        ItemStack.EMPTY,
+                                        changedSlots));
+                    }
+                    case 0 -> McUtils.sendMessageToClient(
+                            new TranslatableComponent("feature.wynntils.emeraldPouchKeybind.noPouch")
+                                    .withStyle(ChatFormatting.DARK_RED));
+
+                    case 1 -> {
+                        int slotNumber = emeraldPouches
+                                .entrySet()
+                                .iterator()
+                                .next()
+                                .getKey(); // Just get the first value in the HashMap since we only have one value
+
+                        if (slotNumber < 9) {
+                            // sendPacket uses raw slot numbers, we need to remap the hotbar
+                            slotNumber += 36;
+                        }
+                        changedSlots.putIfAbsent(
+                                slotNumber, player.getInventory().getItem(slotNumber));
+                        McUtils.player()
+                                .connection
+                                .send(new ServerboundContainerClickPacket(
+                                        player.inventoryMenu.containerId,
+                                        player.inventoryMenu.getStateId(),
+                                        slotNumber,
+                                        1,
+                                        ClickType.PICKUP,
+                                        ItemStack.EMPTY,
+                                        changedSlots));
+                    }
+                }
+            });
+
+    private int getPouchUsage(ItemStack stack) { // TODO: move to EmeraldPouchManager or equivalent when created
+        String lore = ItemUtils.getStringLore(stack);
+        Matcher usageMatcher = POUCH_USAGE_PATTERN.matcher(lore);
+        if (!usageMatcher.find()) {
+            if (lore.contains("§7Empty")) {
+                return 0;
+            }
+
+            return -1;
+        }
+        return Integer.parseInt(usageMatcher.group(1).replaceAll("\\s", ""));
+    }
+
+    private int getPouchCapacity(ItemStack stack) { // TODO: move to EmeraldPouchManager or equiv when created
+        String lore = ItemUtils.getStringLore(stack);
+        Matcher capacityMatcher = POUCH_CAPACITY_PATTERN.matcher(lore);
+        if (!capacityMatcher.find()) {
+            return -1;
+        }
+        int capacity = Integer.parseInt(capacityMatcher.group(1)) * 64;
+        if (capacityMatcher.group(2).equals(EmeraldSymbols.LE)) capacity *= 64;
+        if (capacityMatcher.group(2).equals("stx")) capacity *= 4096;
```

Comment:
this parsing of emerald count should be generalized to likely some external method